### PR TITLE
animate hearts around pet after feeding

### PR DESCRIPTION
**Pull request by minimatime**
Issue Number(s): #131 

What does this PR change and why?

### Checklist

- [ ] Requirements have been implemented
- [ ] Acceptance criteria is met
- [ ] Database schema docs have been updated or are not necessary
- [ ] Code follows design and style guidelines
- [ ] Commits follow guidelines (concise, squashed, etc)
- [ ] Relevant reviewers (Senior Dev/EM/Designers) have been assigned to this PR

### Critical Changes

- Database change/migration to run
- Environment config change
- Breaking API change

### Related PRs

- #number_of_pr

### Testing

Enumerate steps to test the functionality of your ticket. This should include edge cases and testing of error handling, if applicable.

### CODE REVIEW

File: src/components/modals/LevelUpModal.tsx [hunk 1]
```diff
@@ -102,11 +102,11 @@ export default function FoodModal({
...
                 width={0}
                 height={0}
-                sizes="(max-width: 768px) 60vw, (max-width: 1200px) 50vw, 450px"
+                sizes="(max-width: 768px) 80vw, (max-width: 1200px) 70vw, 600px"
                 alt="Shine"
-                className="absolute top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2 mx-auto mb-4 w-auto h-[60vh] tablet:h-[50vh] tiny:h-[55vh]"
+                className="absolute top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2 w-full max-w-[100%] h-auto"
               />
-              <div className="absolute top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2 flex items-center justify-cente h-10">
+              <div className="absolute top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2 flex items-center justify-center h-10">
```
[the remark this answers is not in the document]
Comment (minimatime) [reply src/components/modals/LevelUpModal.tsx]: ok some of my changes didn't get pushed i see

File: src/components/ui/Heart.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    }, 2000);
+
+    return () => clearTimeout(timer);
+  }, [onComplete]);
+
+  if (!isVisible) return null;
+
+  return (
+    <div className="absolute inset-0 pointer-events-none z-40">
+      \
```
[the remark this answers is not in the document]
Comment (minimatime) [reply src/components/ui/Heart.tsx]: done

File: src/components/ui/Bubble.tsx [hunk 3]
```diff
@@ -5,19 +5,21 @@ interface BubbleProps {
...
-      <div className="w-full mobile:p-4 desktop:p-8 bg-white tiny:text-xl short:text-2xl mobile:text-md tablet:text-xl desktop:text-3xl largeDesktop:text-4xl font-bold text-[#343859] text-center shadow-bubble font-quantico">
-        {text}
-      </div>
-      <div className="h-10 w-10 ml-12 relative">
-        <Image
-          src="/misc/chatBubbleCorner.svg"
-          alt="Chat Bubble Decoration"
-          fill
-        />
+    <div className="relative w-96">
```
[the remark this answers is not in the document]
Comment (rgodha24) [reply src/components/ui/Bubble.tsx]: this should be fine

File: src/components/ui/Bubble.tsx [hunk 4]
```diff
@@ -5,19 +5,21 @@ interface BubbleProps {
...
+      <Image
+        src="/misc/messagebubble.svg"
+        alt="Speech Bubble"
+        width={600}
+        height={400}
+      />
+
+      <div className="absolute top-[15%] left-[5%] right-[5%] bottom-[25%] flex items-center justify-center px-4">
+        <p className="text-2xl font-bold text-[#343859] text-center font-quantico leading-tight">
```
[the remark this answers is not in the document]
Comment (rgodha24) [reply src/components/ui/Bubble.tsx]: i think this is real tho

File: src/components/ui/Heart.tsx [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+        <Image
+          src="/misc/heart.svg"
+          alt="Heart"
+          width={60}
+          height={60}
+          className="w-auto h-auto"
+        />
+      </div>
+      <style jsx>{`
+        @keyframes float-heart-left {
```
Comment (rgodha24) [inline src/components/ui/Heart.tsx]: can you move these to `src/styles/globals.css` pls

**Review by rgodha24 — CHANGES_REQUESTED** [review]
this is sooo awesome i love it so much. just one tiny change

https://github.com/user-attachments/assets/1d15d9a9-3dfe-4be1-8d9d-2cbcc5ff7459

**Review by rgodha24 — APPROVED** [review]
nice
